doc/install/install-from-source.md

```diff
@@ -14,45 +14,74 @@ cd deepmd-kit
 deepmd_source_dir=`pwd`
 ```
 
-## Install the python interface
-### Install Tensorflow's python interface
-First, check the python version on your machine.
+## Install the Python interface
+### Install Backend's Python interface
+First, check the Python version on your machine.
 Python 3.8 or above is required.
 ```bash
 python --version
 ```
 
-We follow the virtual environment approach to install TensorFlow's Python interface. The full instruction can be found on the official [TensorFlow website](https://www.tensorflow.org/install/pip). TensorFlow 2.2 or later is supported. Now we assume that the Python interface will be installed to the virtual environment directory `$tensorflow_venv`
+We follow the virtual environment approach to install the backend's Python interface.
+Now we assume that the Python interface will be installed in the virtual environment directory `$deepmd_venv`:
+
 ```bash
-virtualenv -p python3 $tensorflow_venv
-source $tensorflow_venv/bin/activate
+virtualenv -p python3 $deepmd_venv
+source $deepmd_venv/bin/activate
 pip install --upgrade pip
+```
+
+::::{tab-set}
+
+:::{tab-item} TensorFlow {{ tensorflow_icon }}
+
+The full instruction to install TensorFlow can be found on the official [TensorFlow website](https://www.tensorflow.org/install/pip). TensorFlow 2.2 or later is supported.
+```bash
 pip install --upgrade tensorflow
 ```
-It is important that every time a new shell is started and one wants to use `DeePMD-kit`, the virtual environment should be activated by
+
+If one does not need the GPU support of DeePMD-kit and is concerned about package size, the CPU-only version of TensorFlow should be installed by
 ```bash
-source $tensorflow_venv/bin/activate
+pip install --upgrade tensorflow-cpu
 ```
-if one wants to skip out of the virtual environment, he/she can do
+
+To verify the installation, run
 ```bash
-deactivate
+python -c "import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000])))"
 ```
-If one has multiple python interpreters named something like python3.x, it can be specified by, for example
+
+One can also [build the TensorFlow Python interface from source](https://www.tensorflow.org/install/source) for customized hardware optimization, such as CUDA, ROCM, or OneDNN support.
+
+:::
+
+:::{tab-item} PyTorch {{ pytorch_icon }}
+
```

Comment:
it seems some of the special characters are not rendered correctly.

Reply:
I can see it rendered correctly in the preview of the web page. If you mean the page rendered in GitHub, we have deprecated that one.

![image](https://github.com/deepmodeling/deepmd-kit/assets/9496702/9a15fe44-b245-4cd3-8930-7a65c2861acf)

Reply:
Approved.